AUTHORS

```diff
@@ -0,0 +1,935 @@
+Thomas Tuts <thomas.tuts@gmail.com>
+Thorben Egberts <tegberts@gmail.com>
+Tiago Danin <TiagoDanin@outlook.com>
+TiagoDanin <TiagoDanin@outlook.com>
+TiagoDanin <Tiagojdanin@outlook.com>
+Tim Fish <tim@timfish.uk>
+Tim Fish <tim@timfishy.co.uk>
+Tim Ruffles <timruffles@googlemail.com>
+Timothy Cyrus <tcyrus@users.noreply.github.com>
+TingPing <tingping@tingping.se>
+Todd Wolfson <todd@twolfson.com>
+Toinane <toinane@crea-that.fr>
+Tom Floyer <tomfloyer@users.noreply.github.com>
+Tom Hughes <tom@airtime.com>
+Tom Moor <tom.moor@gmail.com>
+Tom Watson <tom.james.watson@googlemail.com>
+Tomas Rycl <torycl@microsoft.com>
+Tomáš Hanáček <tomas.hanacek1@gmail.com>
+Tony Ganch <tonyganch@gmail.com>
+Toon Borgers <borgers.toon@gmail.com>
+Travis Tilley <ttilley@gmail.com>
+Tse Kit Yam <tsekityam@users.noreply.github.com>
+TyanNN <easylikerainbow@gmail.com>
+Tyler Gibson <tgibson@microsoft.com>
+Tyler Gibson <tyler@snotmonkey.com>
+VReality64 <VReality64@gmail.com>
+Vadim Macagon <vadim.macagon@gmail.com>
+Vanessa Yuen <6842965+vanessayuenn@users.noreply.github.com>
+Vanessa Yuen <vanessa.yh.yuen@gmail.com>
+Varun Agrawal <Varun@VarunAgw.com>
+Vasu Mahesh <vasu.mahesh@hotmail.com>
+Veck Hsiao <fbukevin@gmail.com>
+Victor <vb4007@gmail.com>
+Victor Zhu <zzy809@126.com>
+Vijay Pushkin <vijaypushkin.a@gmail.com>
+Vincent Quagliaro <vincentq@theodo.fr>
+Vishesh Handa <vishesh.handa@eyeos.com>
+Vitalie Lazu <vitalie.lazu@gmail.com>
+Vivek Patel <vvivek.patel@gmail.com>
+Vjekoslav Ratkajec <vjeko@labtiva.com>
+Vladimir Krivosheev <develar@gmail.com>
+Vladimir Vainer <str3l0k@gmail.com>
+Volodymyr Bezuglyy <vbez@isd.dp.ua>
+Vítor Galvão <info@vitorgalvao.com>
+W L Bentley <william@wlbentley.net>
+WangYunjian <1124926655@qq.com>
+Wayne Loopy <sumel014@naver.com>
+Wei Wang <weizhi.station.wang@gmail.com>
+Weiqiang Lin <23leo@163.com>
+Weiqiang Lin <eoukan@gmail.com>
+Wen Luo <vaenow@users.noreply.github.com>
+WenryXu <wenryxu@outlook.com>
+Will Eccles <WillEccles@users.noreply.github.com>
+Will Tange <will@brt.sexy>
+Wilson Page <wilsonpage@me.com>
+Wliu <50Wliu@users.noreply.github.com>
+Wolfgang <wolfgang42@users.noreply.github.com>
+Wujg <neutroustroup@gmail.com>
+Xaver Hellauer <software@hellauer.bayern>
+XiYang <me@yunzhenhuan.com>
+Xue Fuqiao <xfq.free@gmail.com>
+Yarkhan <rasrpg@gmail.com>
+Yassine Laassel <ylaassel@me.com>
+Yasunari Fujieda <edamame1224@gmail.com>
+Yeechan Lu <wz.bluesnow@gmail.com>
+Yegun Kim <yegunkim@uwm.edu>
+Yifei Teng <yifei@skycatch.com>
+Yonggang Luo <luoyonggang@gmail.com>
+Yongmin Hong <revi@pobox.com>
+Yoshua Wuyts <yoshuawuyts@gmail.com>
+Yoyo Zhou <yozhov@gmail.com>
+Yudu Ban <banyudu@gmail.com>
+Yui <yuit@users.noreply.github.com>
+Yury Solovyov <biohazard707@gmail.com>
+Yuya Ochiai <yuya0321@gmail.com>
+Zachary Flower <zach@ninjaninja.net>
+Zachary Flower <zach@zacharyflower.com>
+Zeke "Offline in April" Sikelianos <zeke@github.com>
+Zeke Sikelianos <zeke@github.com>
+Zeke Sikelianos <zeke@sikelianos.com>
```

Comment:
@zeke which of these three emails would you like to use as your canonical one?

Reply:
`Zeke Sikelianos <zeke@sikelianos.com>` :)